### PR TITLE
Is this it??

### PR DESCRIPTION
**Pull request by elevin72**

### CODE REVIEW

File: src/main/rendering/rayTracing/BasicRayTracer.java [hunk 1]
```diff
@@ -20,13 +22,15 @@
...
 
 	@Override
 	public Colour trace(Ray ray) {
-		// TODO: implement
-		return null;
+		List<Point> intersections = scene.geometries.intersect(ray);
+		if (intersections.isEmpty()) {
+			return scene.background;
+		} 
+		return scene.ambient.colour;
```
Comment (abrahammurciano) [inline src/main/rendering/rayTracing/BasicRayTracer.java:34]: ternary op is cool, this wld be a nice place to use it lol

File: src/main/rendering/rayTracing/BasicRayTracer.java [hunk 2]
```diff
@@ -20,13 +22,15 @@
 	 */
 	public BasicRayTracer(Scene scene) {
 		super(scene);
-		// TODO: implement
+		// TODO: implement. Is anything else necessary here?
```
Comment (abrahammurciano) [inline src/main/rendering/rayTracing/BasicRayTracer.java:25]: this is it, yes

**Review by abrahammurciano — APPROVED** [review]
Basic: (adjective) involving little substance; not complex;

Yes, that is it.

**Comment by abrahammurciano** [comment]
I believe tests need to be added too

File: src/main/rendering/rayTracing/BasicRayTracer.java (hunk 1 above)
Comment (elevin72) [reply src/main/rendering/rayTracing/BasicRayTracer.java:34]: Why do you think they're cool?

**Pull request merged**